app/src/main/java/com/sun_asterisk/moviedb_50/data/source/remote/Repository.kt

```diff
@@ -1,22 +1,23 @@
 package com.sun_asterisk.moviedb_50.data.source.remote
 
-import com.sun_asterisk.moviedb_50.data.model.Genres
-import com.sun_asterisk.moviedb_50.data.model.Movie
+import com.sun_asterisk.moviedb_50.data.model.GenresResponse
+import com.sun_asterisk.moviedb_50.data.model.MoviesResponse
 import com.sun_asterisk.moviedb_50.data.source.DataSource
 
 class Repository private constructor(
     private val remoteDataSource: DataSource.RemoteDataSource,
     private val localDataSource: DataSource.LocalDataSource
 ) {
-    fun getGenres(onFetchDataJsonListener: OnFetchDataJsonListener<MutableList<Genres>>?) {
+    fun getGenres(onFetchDataJsonListener: OnFetchDataJsonListener<GenresResponse>?) {
```

Comment:
rõ ràng từ Presenter, callback truyền vào luôn không null thì tại sao ở đây lại phải để là nullable nhỉ? 
```suggestion
    fun getGenres(listener: OnFetchDataJsonListener<GenresResponse>) {
```